src/ERC20Bridgeable/libraries/LibERC20Bridgeable.sol

```diff
@@ -0,0 +1,175 @@
+library LibERC20Bridgeable {
+    /// @notice Revert when a provided receiver is invalid(e.g,zero address) .
+    /// @param _receiver The invalid reciever address.
+    error ERC20InvalidReciever(address _receiver);
+
+    /// @notice Thrown when the sender address is invalid (e.g., zero address).
+    /// @param _sender The invalid sender address.
+    error ERC20InvalidSender(address _sender);
+
+    /// @notice Revert when caller is not a trusted bridge.
+    /// @param _caller The unauthorized caller.
+    error ERC20InvalidBridgeAccount(address _caller);
+
+    // @notice Revert when caller address is invalid.
+    /// @param _caller is the invalid address.
+    error ERC20InvalidCallerAddress(address _caller);
+
+    /// @notice Thrown when the account does not have a specific role.
+    /// @param _role The role that the account does not have.
+    /// @param _account The account that does not have the role.
+    error AccessControlUnauthorizedAccount(address _account, bytes32 _role);
+    
+    error ERC20InsufficientBalance(address _from, uint256 _accountBalance, uint256 _value);
+
+    /// @notice Emitted when tokens are minted via a cross-chain bridge.
+    /// @param _to The recipient of minted tokens.
+    /// @param _amount The amount minted.
+    /// @param _sender The bridge account that triggered the mint (msg.sender).
+    event CrosschainMint(address indexed _to, uint256 _amount, address indexed _sender);
+
+    /// @notice Emitted when a crosschain transfer burns tokens.
+    /// @param _from     Address of the account tokens are being burned from.
+    /// @param _amount   Amount of tokens burned.
+    /// @param _sender   Address of the caller (msg.sender) who invoked crosschainBurn.
+    event CrosschainBurn(address indexed _from, uint256 _amount, address indexed _sender);
+
+    /// @notice Emitted when tokens are transferred between two addresses.
+    /// @param _from Address sending the tokens.
+    /// @param _to Address receiving the tokens.
+    /// @param _value Amount of tokens transferred.
+    event Transfer(address indexed _from, address indexed _to, uint256 _value);
+
+    /// -----------------------------------------------------------------------
+    /// ERC20 integration (re-uses ERC20Facet storage layout)
+    /// -----------------------------------------------------------------------
+
+    /// @notice Storage slot for ERC-20 token using ERC8042 for storage location standardization
+    /// @dev Storage position determined by the keccak256 hash of the diamond storage identifier.
+    bytes32 constant ERC20_STORAGE_POSITION = keccak256("compose.erc20");
+
+    /**
+     * @dev ERC-8042 compliant storage struct for ERC20 token data.
+     * @custom:storage-location erc8042:compose.erc20
+     */
+    struct ERC20Storage {
+        string name;
+        string symbol;
+        uint8 decimals;
+        uint256 totalSupply;
+        mapping(address owner => uint256 balance) balanceOf;
+    }
```

Comment:
The order of variables in ERC20Storage from the `ERC20Facet` has changed.  Please update.